HQSmokeTests/utilities/email_pytest_report.py

```diff
@@ -0,0 +1,163 @@
+"""
+Qxf2 Services: Utility script to send pytest test report email
+* Supports both text and html formatted messages
+* Supports text, html, image, audio files as an attachment
+
+To Do:
+* Provide support to add multiple attachment
+
+Note:
+* We added subject, email body message as per our need. You can update that as per your requirement.
+* To generate html formatted test report, you need to use pytest-html plugin. To install it use command: pip install pytest-html
+* To generate pytest_report.html file use following command from the root of repo e.g. py.test --html = log/pytest_report.html
+* To generate pytest_report.log file use following command from the root of repo e.g. py.test -k example_form -r F -v > log/pytest_report.log
+"""
+import datetime
+import smtplib
+import os,sys
+from email.mime.text import MIMEText
+from email.mime.image import MIMEImage
+from email.mime.audio import MIMEAudio
+from email.mime.multipart import MIMEMultipart
+from email.mime.base import MIMEBase
+import mimetypes
+from email import encoders
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+import HQSmokeTests.utilities.email_conf as conf_file
+
+
+class Email_Pytest_Report:
```

Comment:
I think this could be accomplished with the `send-email` action: https://github.com/marketplace/actions/send-email